docs: add changelog entry for infrastructure scaling improvements (#2516)
langfuse/langfuse-docs: pages/docs/v4.mdx

---
title: Langfuse Preview (fast)
description: Langfuse now delivers faster product performance. Enable the preview and upgrade SDKs to get the new experience without delays.
---

import { Callout } from "nextra/components";
import { BarChart3, Book, Github, Zap } from "lucide-react";

# Langfuse Preview (fast)

Langfuse now delivers faster product performance at scale. This preview brings faster charts, filters, APIs, and evaluation workflows to Langfuse Cloud.

We will keep this page updated as we roll out additional related features and share OSS and self-hosted support updates.

## Get access to the preview

<Steps>

### Enable the beta UI experience

<Frame className="max-w-xs mx-auto">
  ![Beta toggle in Langfuse UI](/images/changelog/2026-03-10-v4-preview/v4-beta-togle.png)
</Frame>

In the UI, a beta toggle is available to opt into the new experience. Not all screens have been migrated to the new data model yet.

<Callout type="info">

> Data from older SDK versions can be delayed by up to 10 minutes in the new UI. Upgrade to the latest SDK versions to see your data in real time.

Reviewer: ```suggestion
Data from SDKs `langfuse-js` < `5.0.0` and `langfuse-python` < `4.0.0` is delayed for up to 10 minutes in the new UI. Upgrade to the latest SDK versions to see your data in real time.
```
